tests/suites/1_stateful/00_copy/00_0000_copy_from_stage.sh

```diff
@@ -30,10 +31,12 @@ copy_from_stage_cases=(
   "copy into ontime200 from @s1 FILES = ('ontime_200.csv.zst') FILE_FORMAT = (type = 'CSV' field_delimiter = ',' compression = 'zstd'  record_delimiter = '\n' skip_header = 1);"
   # copy bz2 csv
   "copy into ontime200 from @s1 FILES = ('ontime_200.csv.bz2') FILE_FORMAT = (type = 'CSV' field_delimiter = ',' compression = 'bz2'  record_delimiter = '\n' skip_header = 1);"
-   # copy xz csv
-    "copy into ontime200 from @s1 FILES = ('ontime_200.csv.xz') FILE_FORMAT = (type = 'CSV' field_delimiter = ',' compression = 'xz'  record_delimiter = '\n' skip_header = 1);"
+  # copy xz csv
+  "copy into ontime200 from @s1 FILES = ('ontime_200.csv.xz') FILE_FORMAT = (type = 'CSV' field_delimiter = ',' compression = 'xz'  record_delimiter = '\n' skip_header = 1);"
```

Comment:
have to set field_delimiter,  record_delimiter for now?

Reply:
No, it's just style fixed.